docs/pbc/all/merchant-management/202407.0/marketplace/marketplace-merchant-commission-feature-overview.md

```diff
@@ -0,0 +1,192 @@
+
+For example, your setup with groups and priority could be as follows. We simplified the commissions for this example by removing irrelevant fields.
+
+| Commission Key | Priority | Group |
+| - | - | - |
+| MC01 | 1 | Primary |
+| MC02 | 2 | Primary |
+| MC03 | 2 | Secondary |
+| MC04 | 1 | Secondary |
+
+If an order item fulfills the conditions of all the commissions in the prior example, commissions `MC01` and `MC04` are applied. First, the system selects commission `MC01` as the commission in the primary group with the highest priority. Then, the system selects commission `MC04` as the commission in the secondary group with the highest priority.
+
+
+
+
+
+
+
+
+Apply commission to GROSS or NET price
+This Merchant Commission Price Mode configuration controls which prices are used as a basis to calculate commission: Gross Price or Net Price.
+
+The item Gross Price and Net Price is configured in the backend (including volume pricing) and does not include any discounts applied afterward in the shopping cart.
+
+Its defined by MERCHANT_COMMISSION_PRICE_MODE_PER_STORE, see https://github.com/spryker/spryker-docs/blob/master/_includes/pbc/all/install-features/202407.0/marketplace/install-the-marketplace-merchant-commission.md#2-set-up-configuration
+This configuration is different from an order price mode and defines behavior for the whole store.
+
+## Current limitations
+
+Price Mode switcher is not supported. The Merchant Commission Price Mode must match the default price mode of the store that sets it for the sales order as a result. Otherwise, the percentage commission will be set to 0 for all items in the order.
+
+Supported configuration:
+
+| Order Price Mode | Merchant Commission Price Mode |
+| - | - |
+| GROSS_MODE | GROSS_MODE |
+| NET_MODE | NET_MODE |
+
+
+
+## Applying merchant commissions to the sum item total price to pay after discounts with additions
+
+You might want to change how commission is calculated on the project level by applying commissions to the sum item total price to pay after discounts with additions. In Glue API, this is represented by `sumPriceToPayAggregation`. This can be useful when you want to apply commissions to total paid by a customer a line item. This amount is what the customer is getting charged. Sum of all line items plus expenses, like a shipment fee, is equal to the grand total.
+
+This customization is factored into the application design. A developer can change this logic within two hours by following [Create merchant commission calculator type plugin].
+
+Other use cases might be as follows:
+* Apply commission to product price including product options differently
+* Apply commission to special products like bundles or configurable differently
```

Comment:
What do you mean by `differently`?

Reply:
Marketplace operator may choose to have a different way to apply % to product types, for example choose not to charge commission on product options.

Reply:
Do you mean `Differently calculate commission for product price, including product options`?